tests/test_json_output.py

```diff
@@ -0,0 +1,51 @@
+from modules.Connection import Connection
+from BaseCollector import BaseCollector
+import pytest
+import json
+import yaml
+
+
+@pytest.fixture(scope='class')
+def bc_yaml():
+    bc_data = BaseCollector.read_rest_yaml()
+    yield bc_data
+
+
+@pytest.fixture(scope='class')
+def vmon_json():
+    with open("mockingServer/data/vmon.json", 'r') as data:
+        vmon_data = json.load(data)
+    return vmon_data
+
+
+@pytest.fixture(scope='class')
+def logging_json():
+    with open("mockingServer/data/logging.json", 'r') as data:
+        logging_data = json.load(data)
+    return logging_data
+
+
+class TestJsonOuput:
+    def test_yaml_read(self, bc_yaml):
```

Comment:
simple and good, I like it.

Reply:
Thank you very much!

Reply:
I implemented the BaseCollector read in the test_yaml_read now. The fixture should read the data from the file. It then behaves like the other fixtures used in test_json_output.